src/accounts/remove/remove-account-handler.js

```diff
@@ -0,0 +1,202 @@
+const HttpClient = require('../../core/client/http-client');
+const ApiExtract = require('../../core/utils/apiExtract');
+const shouldDeleteDefaultAccountQuestion =
+  ' is your workspace account. Are you sure you want to remove it from Slack?';
+const shouldDeleteAccountWithCurrentChannels = ' is used in these channels:';
+const areYouSure = ' Are you sure you want to remove it from Slack?';
+const shouldDeleteAccount = 'Are you sure you want to remove ';
+
+function getMessageWithButtons(prefix, question, suffix) {
+  return {
+    attachments: [
+      {
+        title: '',
+        text: prefix + question + suffix,
+        callback_id: 'remove-account',
+        attachment_type: 'default',
+        delete_original: true,
+        actions: [
+          {
+            text: 'Remove',
+            value: 'yes',
+            type: 'button',
+            name: 'yes'
+          },
+          {
+            text: 'Cancel',
+            value: 'no',
+            type: 'button',
+            name: 'no'
+          }
+        ]
+      }
+    ]
+  };
+}
+
+class removeAccountHandler {
+  constructor(teamConfigService) {
+    this.teamConfigService = teamConfigService;
+  }
+
+  removeAccount(teamId, channel, userAlias, bot, user) {
+    if (!userAlias) {
+      return this.PromiseToRemove(teamId, channel, userAlias, bot, user);
+    } else {
+      return HttpClient.validateAlias(
+        this.teamConfigService,
+        teamId,
+        userAlias
+      ).then(() => this.PromiseToRemove(teamId, channel, userAlias, bot, user));
+    }
+  }
+
+  PromiseToRemove(teamId, channel, userAlias, bot, user) {
+    let teamConfiguration;
+    let alias = userAlias;
+    return this.teamConfigService
+      .getDefault(teamId)
+      .then(configuration =>
+        !userAlias?HttpClient.validateConfiguration(configuration):configuration)
+      .then(validTeamConfiguration => {
+        teamConfiguration = validTeamConfiguration;
+        if (!alias) {
+          alias = teamConfiguration.getAlias();
+        }
+        return this.teamConfigService.getAccountForChannel(teamId, channel);
+      })
+      .then(accountConfiguration => {
+        if (!userAlias && accountConfiguration) {
+          alias = accountConfiguration.getAlias();
+        }
+        return alias;
+      })
+      .then(alias =>
+        this.ConfirmRemove(teamConfiguration, alias, bot, user, teamId)
+      );
+  }
+
+  ConfirmRemove(teamConfiguration, alias, bot, user, teamId) {
+    let noAlias = alias === undefined;
+    if (noAlias || teamConfiguration.config.alias === alias) {
+      if (noAlias) {
+        alias = 'This';
+      }
+
```

Comment:
all of this: 
ConfirmRemove(teamConfiguration, alias, bot, user, teamId) {
    let noAlias = alias === undefined;
    if (noAlias || teamConfiguration.config.alias === alias) {
      if (noAlias) {
        alias = 'This';
      }
  }
...
}
can be reaplce with : 
 ConfirmRemove(teamConfiguration, alias = 'This', bot, user, teamId) {
....
}